pages/docs/concepts/asyncapi-document/AsyncAPI_structure.md

```diff
@@ -0,0 +1,384 @@
+---
+title: AsyncAPI Structure
+weight: 31
+---
+
```

Comment:
- What is structure in an AsyncAPI document?
- What can I do with it? how does it work?
- what is it's relation to asyncapi spec?